L/libjulia/common.jl

```diff
@@ -189,13 +203,22 @@ function build_julia(ARGS, version)
     # We don't trust the system libm in places
     # So we include a private copy of libopenlibm
     mkdir -p usr/lib
-    cp ${prefix}/lib/libopenlibm.a usr/lib/
+    if [[ "${target}" == *-mingw* ]] && [[ "${version}" == 1.6.* ]]; then
+        cp ${prefix}/bin/libopenlibm.a usr/lib/
+    else
+        cp ${prefix}/lib/libopenlibm.a usr/lib/
```

Comment:
Swap conditions to work in Julia > 1.6 as well... or perhaps just cheat?
```suggestion
        cp ${prefix}/*/libopenlibm.a usr/lib/
```